### ISSUE TITLE
SQL error in /tags/search

### ISSUE THREAD

**Issue by obert01** [MISP/MISP#6475]
# Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Ubuntu
| PHP version               | 7.2
| MISP version / git hash   | 2.4.133

### Expected behavior
Call to "tags/search" should return the list of tags / clusters matching a given value.

### Actual behavior

SQL Exception see below. 

### Steps to reproduce the behavior
```
curl -X GET -o - --header "Authorization: XXXXXX" --header "Accept: application/json" --header "Content-Type: application/json"  \
https://misp_host/tags/search/Emotet
```

### Logs, screenshots, configuration dump, ...

```
2020-10-21 13:54:28 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'GalaxyCluster.value' in 'where clause'
Request URL: /tags/search/Emotet
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(502): PDOStatement->execute(Array)
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(468): DboSource->_execute('SELECT `Tag`.`i...', Array)
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(715): DboSource->execute('SELECT `Tag`.`i...', Array, Array)
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1226): DboSource->fetchAll('SELECT `Tag`.`i...', Array)
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3053): DboSource->read(Object(Tag), Array)
#5 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3025): Model->_readDataSource('all', Array)
#6 /var/www/MISP/app/Controller/TagsController.php(1155): Model->find('all', Array)
#7 [internal function]: TagsController->search(Array)
#8 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(499): ReflectionMethod->invokeArgs(Object(TagsController), Array)
#9 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#10 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(TagsController), Object(CakeRequest))
#11 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
```

**Comment by mokaddem** [comment]
The actual URL for searching tags is the following `/tags/index/searchall:{tag_name}`, so for example:
`/tags/index/searchall:amber`

**Comment by obert01** [comment]
Hello,

The entry point /tags/index/searchall unfortunately does not allow to search in galaxy/clusters, which is what I am actually lookign for. 

With "/tags/search/Emotet", I was able to find the tag misp-galaxy:malpedia="Geodo" (which has Emotet as synonym), while it is not possible with /tags/index/searchall.

**Comment by mokaddem** [comment]
Hey, It is now fixed on 2.4.
Feel free to update and thanks for the report!
